### PR TITLE
Upgrade andes client

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**
## Purpose
Upgrade andes client & pack secure vault lib, since It's needed for andes client.

## Goals
Fix https://github.com/wso2/product-ei/issues/1637

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -256,6 +256,13 @@
                                     <overWrite>true</overWrite>
                                     <outputDirectory>target/ext/broker</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.wso2.securevault</groupId>
+                                    <artifactId>org.wso2.securevault</artifactId>
+                                    <version>1.0.0-wso2v2</version>
```
Comment (Manuri) [inline distribution/pom.xml]: Can we use a property for this?

Comment (bsenduran) [inline distribution/pom.xml]: Shall we extract the version to a property

Comment (madhawa-gunasekara) [reply distribution/pom.xml]: :+1:  done

Comment (madhawa-gunasekara) [reply distribution/pom.xml]: :+1:  done

**Pull request merged**